Dcat editorial improvements

riccardoAlbertoni opened the pull request:
A draft updating initial sections to make them more consistent with DCAT 3.

Please check if anything is missing, and feel free to comment and change.

preview: https://raw.githack.com/w3c/dxwg/dcat-editorialimprovements/dcat/index.html
diff: https://services.w3.org/htmldiff?doc1=https://w3c.github.io/dxwg/dcat/&doc2=https://raw.githack.com/w3c/dxwg/dcat-editorialimprovements/dcat/index.html

riccardoAlbertoni committed:
updating  initial editorial note and  note for conformance, making it consistent with DCAT 3

riccardoAlbertoni committed:
updating till DCAT history

riccardoAlbertoni committed:
updating till section namespaces, removing redundant comments, tweaking some of the previous commits

riccardoAlbertoni committed:
tweaking sections

andrea-perego commented on dcat/index.html:
```diff
@@ -46,37 +46,34 @@
 
     <p>
-        This document defines a major revision of the original DCAT vocabulary ([[?VOCAB-DCAT-20140116]]) in response to new use cases, requirements and community experience since that publication.  This revision extends the original DCAT standard in line with community practice while supporting diverse approaches to data description and dataset exchange. The main changes to the DCAT vocabulary h ...
+        This document defines a major revision of the DCAT 2 vocabulary ([[?vocab-dcat-2]]) in response to use cases, requirements and community experience which could not considered during the previous vocabulary development.  This revision extends the original DCAT standard in line with community practice while supporting diverse approaches to data description and dataset exchange. The main cha ...
```
```suggestion
        This document defines a major revision of the DCAT 2 vocabulary ([[?VOCAB-DCAT-2]]) in response to use cases, requirements and community experience which could not considered during the previous vocabulary development.  This revision extends the original DCAT standard in line with community practice while supporting diverse approaches to data description and dataset exchange. The main changes to the DCAT vocabulary have been: 
```

andrea-perego commented on dcat/index.html:
```diff
@@ -151,13 +147,13 @@ <h2>Introduction</h2>
-    For example, in DCAT 2 we provide classes, properties and guidance to address <a href="#dereferenceable-identifiers">identifiers</a>, <a href="#quality-information">dataset quality information</a>, and <a href="#data-citation">data citation</a> issues.</p>
-    <p>This revision includes re-writing of the specification throughout. Significant changes from the 2014 Recommendation are marked within the text using "Note" sections, as well as being described in <a href="#changes"></a>.</p>
+    <p>The original Recommendation [[?VOCAB-DCAT-20140116]] published in January 2014 provided the basic framework for describing datasets. It made an important distinction between a <i>dataset</i> as an abstract idea and a <i>distribution</i> as a manifestation of the dataset. Although DCAT has been widely adopted, it has become clear that the original specification lacked a number of essential  ...
+    For example, in DCAT 2 provided classes, properties and guidance to address <a href="#dereferenceable-identifiers">identifiers</a>, <a href="#quality-information">dataset quality information</a>, and <a href="#data-citation">data citation</a> issues.</p>
```
```suggestion
    For example, DCAT 2 provided classes, properties and guidance to address <a href="#dereferenceable-identifiers">identifiers</a>, <a href="#quality-information">dataset quality information</a>, and <a href="#data-citation">data citation</a> issues.</p>
```

andrea-perego commented on dcat/index.html:
```diff
@@ -46,37 +46,34 @@
     </p>
     <p>
-        The exit criteria for CR focussed on v2 new features that replicate features that were included in application profiles of v1 as a way of remedying missing and necessary elements.  The exit criteria also included recent commitments by organisations such as EC Joinup to <a href='https://joinup.ec.europa.eu/solution/abr-specification-registry-registries/document/specification-registry-regis ...
+        The exit criteria for CR focussed on v3 new features that replicate features that were included in application profiles of v2 as a way of remedying missing and necessary elements.  The exit criteria also included recent commitments by organisations such as EC Joinup to <a href='https://joinup.ec.europa.eu/solution/abr-specification-registry-registries/document/specification-registry-regis ...
```
```suggestion
        The exit criteria for CR focussed on v3 new features that replicate features that were included in application profiles of v2 as a way of remedying missing and necessary elements.  <!--The exit criteria also included recent commitments by organisations such as EC Joinup to <a href='https://joinup.ec.europa.eu/solution/abr-specification-registry-registries/document/specification-registry-registries-version-meeting-september'>adopt the DCAT v2 model</a> in their work.-->
```

andrea-perego replied:
As far as I know, there are not yet explicit commitments to update DCAT-AP wrt to DCAT3, although this is likely to happen.

dr-shorthair reviewed (commented):
OK so far, with Andrea's proposed changes.

riccardoAlbertoni committed:
Update dcat/index.html

Co-authored-by: Andrea Perego <andrea-perego@users.noreply.github.com>

riccardoAlbertoni committed:
Update dcat/index.html

Co-authored-by: Andrea Perego <andrea-perego@users.noreply.github.com>

riccardoAlbertoni committed:
Update dcat/index.html

Co-authored-by: Andrea Perego <andrea-perego@users.noreply.github.com>

davebrowning reviewed (commented):
There are a lot of NOTEs that reference "DCAT 2" through the body of the text which also need tweaking or removing - my instinct is that most of them can go, but perhaps we should discuss/agree that that's appropriate.

The changes to the early parts of the document look fine to me

riccardoAlbertoni commented:
@davebrowning, are you referring to the notes saying when the terms were added?  
I was thinking to leave them. Because I guess these notes might help in different situations, e.g., when trying to understand with what version a dataset is compliant. 

I've noticed that some of these notes also explain why the term was added and not only when. I would be keen to keep also those as, in my opinion, they help to understand the context of the choice we made and the context is often useful to better use of the standard.

As for things we might consider reviewing,  the section  6.1 RDF representation is not very aligned with dcat 3.
-Do we want to add a note saying "a new RDF will be made available for DCAT 3 as soon as the vocabulary is finalized"?

davebrowning commented:
Hi @riccardoAlbertoni - yes, the notes that say that they were added in DCAT2 - everything else was in the initial version, of course. As you say, sometimes it includes something of the rationale behind the addition, and that's definitely worth preserving in some way.  I think we maybe want to: 

1. Provide a consistent way to highlight 'design' notes (the "why" it was done that way).  That could be a modified form of the NOTE format.

2. Something that tracks when a term was introduced (and I guess, any other status change such as deprecation.....if we ever did that).  As it is, this is NOTEd in the section 6 definitions and acknowledged in the Appendix D Change History. It probably has to be in both but I was wondering if a table in its own chapter as well might actually be more useful. I think it deserves a given a bit more prominence. We could use the existing style to highlight any new terms introduced in this recommendation (ie DCAT 3).   I haven't had a chance to look for any prior art in any other recommendation that might have had to do this - will try to find time to do that in the next few days & see if I can sketch out an approach and see what we all think.

As for the section 6.1 RDF - yeah, I saw that too.  I agree that we should tag it with a note as you suggest for now.

riccardoAlbertoni committed:
adding editorial note saying we will provide new RDF as soon as the vocabulary is finalized

riccardoAlbertoni committed:
adding a note on current discussion for series

riccardoAlbertoni commented:
added the note on RDF serializations, let's consider if we want to include also the note added via PR #1439

pwin committed:
Update index.html

Nuancing the text about the discussion to ensure that we don't end up splitting the discussion across multiple issues

pwin committed:
Merge pull request #1439 from w3c/dcat-editorialimprovementsBis

Dcat editorial improvements bis

riccardoAlbertoni commented:
As discussed yesterday in the call, points 1 and point 2 in Dave's message will be developed and implemented in a new PR.
We merge the improvements we have so far, to include them in the upcoming  WD.
